static/src/javascripts/projects/common/views/svgs.js

```diff
@@ -0,0 +1,96 @@
+import marketSameIcon from 'svg-loader!svgs/icon/market-same.svg';
+import arrowicon from 'svg-loader!svgs/icon/arrow.svg';
+import arrowdownicon from 'svg-loader!svgs/icon/arrow-down.svg';
+import crossIcon from 'svg-loader!svgs/icon/cross.svg';
+import quoteIcon from 'svg-loader!svgs/icon/quote.svg';
+import paidContent from 'svg-loader!svgs/commercial/paid-content.svg';
+import closeCentralIcon from 'svg-loader!svgs/icon/close-central.svg';
+import arrowWhiteRight from 'svg-loader!svgs/icon/arrow-white-right.svg';
+import arrowRight from 'svg-loader!svgs/icon/arrow-right.svg';
+import bookmark from 'svg-loader!svgs/icon/bookmark.svg';
+import dropdownMask from 'svg-loader!svgs/icon/dropdown-mask.svg';
+import commentAnchor from 'svg-loader!svgs/icon/comment-anchor.svg';
+import reply from 'svg-loader!svgs/icon/reply.svg';
+import expandImage from 'svg-loader!svgs/icon/expand-image.svg';
+import cursor from 'svg-loader!svgs/icon/cursor.svg';
+import plus from 'svg-loader!svgs/icon/plus.svg';
+import share from 'svg-loader!svgs/icon/share.svg';
+import shareTwitter from 'svg-loader!svgs/icon/share-twitter.svg';
+import shareEmail from 'svg-loader!svgs/icon/share-email.svg';
+import shareFacebook from 'svg-loader!svgs/icon/share-facebook.svg';
+import sharePinterest from 'svg-loader!svgs/icon/share-pinterest.svg';
+import shareGPlus from 'svg-loader!svgs/icon/share-gplus.svg';
+import externalLink from 'svg-loader!svgs/icon/external-link.svg';
+import tick from 'svg-loader!svgs/icon/tick.svg';
+import notificationsOff from 'svg-loader!svgs/icon/notification-off.svg';
+import notificationsOn from 'svg-loader!svgs/icon/notification-on.svg';
+import glabsLogoSmall from 'svg-loader!svgs/logo/glabs-logo-small.svg';
+import membershipLogo from 'svg-loader!svgs/commercial/logo-membership.svg';
+import adblockCoins from 'svg-loader!svgs/commercial/adblock-coins.svg';
+import notificationsExplainerDesktop
+    from 'svg-loader!svgs/notifications-explainer-desktop.svg';
+import notificationsExplainerMobile
+    from 'svg-loader!svgs/notifications-explainer-mobile.svg';
+import adblockCoinsUS from 'svg-loader!svgs/commercial/adblock-coins-us.svg';
+import star from 'svg-loader!svgs/icon/star.svg';
+import chevronRight from 'svg-loader!svgs/icon/chevron-right.svg';
+
+import { addClassesAndTitle } from 'common/views/svg';
+
+const svgs = {
+    commentCount16icon,
+    marque36icon,
+    marque54icon,
+    marketDownIcon,
+    marketUpIcon,
+    marketSameIcon,
+    arrowicon,
+    arrowdownicon,
+    crossIcon,
+    quoteIcon,
+    paidContent,
+    closeCentralIcon,
+    arrowWhiteRight,
+    arrowRight,
+    bookmark,
+    dropdownMask,
+    commentAnchor,
+    reply,
+    expandImage,
+    cursor,
+    plus,
+    share,
+    shareTwitter,
+    shareEmail,
+    shareFacebook,
+    sharePinterest,
+    shareGPlus,
+    externalLink,
+    tick,
+    notificationsOff,
+    notificationsOn,
+    glabsLogoSmall,
+    adblockCoinsUk: adblockCoins,
```

Comment:
If you change the import name for `adblockCoinsUS` (into `adblockCoinsUs`) and `membershipLogo` (into `logomembership`) there is no need for the mind-blowing mix anymore. :)

The `adblockCoinsUk` could be a separate variable declaration before that list ...

What do you think?